README.md

```diff
@@ -71,52 +65,41 @@ then configure it.
 npm run web-dev
 ```
 
-Wowser will be served on `http://localhost:8080`.
+Wowser will be served on `http://0.0.0.0:8080/webpack-dev-server/`.
```

Comment:
0.0.0.0 is not an accessible url for windows, I would say: is listening on 0.0.0.0 and you can access locally via 127.0.0.1

Reply:
ok